content/updates/understanding-the-terraform-check-block-feature.md

```diff
@@ -0,0 +1,112 @@
+---
+visible: true
+draft: false
+title: "Understanding the Terraform Check Block Feature"
+author: Veronika Gnilitska
+slug: understanding-the-terraform-check-block-feature
+date: TBD
+description: "We dive into one of Terraform's most recent features to leverage infrastructure validation."
+# image: TBD
+---
+Table of Contents
```

Comment:
We should fine a nice way to generate this -- Did you already do so?

Reply:
I was using this tool [github-markdown-toc](https://github.com/ekalinin/github-markdown-toc#gh-md-toc), but I dug a bit, and this extension seems to work better for our purposes [markdown-all-in-one](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). It updates the TOC is automatically on file save by default.